literature.bib

```diff
@@ -9588,6 +9741,14 @@ @article{TKB+:SCP14
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2013 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@book{HoTT13,
+  author = {The {Univalent Foundations Program}},
+  title = {{Homotopy Type Theory: Univalent Foundations of Mathematics}},
+  publisher = {\url{https://homotopytypetheory.org/book}},
+  address = {Institute for Advanced Study},
```

Comment:
This entry looks severely broken.

Reply:
That is the offical way how this book wants to be cited. It is an open source effort and there is no 💰-publisher behind it so the entry looks weird.

https://homotopytypetheory.org/book/ (scroll down to see bibtex)

Reply:
Ok, thanks for clarifying.